documentation/modules.md

```diff
@@ -142,7 +142,85 @@ modulepath = site:dist:modules:$basemodulepath:.modules
 
 You can learn more about the module changes introduced in Bolt 2.23.0 in the
 [September 2020 developer update](./developer_updates.md#september-2020).
+
+## Migrate a project to use module management
```

Comment:
I don't think we should document this on this page:
- It introduces a task to a page that's really meant to be more of a conceptual overview. 
- It kind of separates the 'before' and 'after' steps for manual migration into 2 pages, which I think might be a bit confusing for users. They'd have to come here to prepare for the migration, and then the module installation page for the next steps. 

I'm torn on where to document this stuff, honestly:
- We could add it to the existing migration stuff on the project page, but it seems weird to have a lot of edge-casey module stuff on there.
- We could add a temporary page for manual migration + module management, and link to it from all of the likely places in the docs. 
- We could add it to the "preparing for 3.0 page," but that presents more complications because we already talk about a different kind of manual migration there. 
- We could add it to the installing modules page under a "migrating" section. 

It's a tough one. I think I'm leaning towards just adding a section to the installing modules page at this point. I guess it doesn't have to be perfect because it's temporary, right? Suggestions welcome.